tests/karate/src/test/resources/karate-config.js

```
@@ -8,36 +8,32 @@ function fn() {
   // Basic configuration
   var config = {
     baseUrl: "http://graphql-engine-test:8080/v1/graphql",
+    webhookUrl: "http://webhook:3001",
```

Biome (2.4.13)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)